src/apis/constants.ts

```diff
@@ -0,0 +1,98 @@
+import { Balance, BalanceOf, BlockNumber, Moment, RuntimeDbWeight, Weight } from "@polkadot/types/interfaces/runtime";
+import { DOT } from "@interlay/polkabtc/interfaces/default";
+import { ApiPromise } from "@polkadot/api";
+import { u32, u64 } from "@polkadot/types/primitive";
+import { Vec } from "@polkadot/types/codec";
+import { WeightToFeeCoefficient } from "@polkadot/types/interfaces/support";
+
+export interface ConstantsAPI {
```

Comment:
If these are constants, maybe we could make them properties instead of functions and implement them as getters?
Let me know what you think.